stablehlo/conversions/tosa/transforms/passes.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2022 OpenXLA Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef MLIR_HLO_TOSA_TRANSFORMS_PASSES_H
```

Comment:
Apologies - didn't notice this during the initial review. Let's rename the header guards to `STABLEHLO_CONVERSIONS_TOSA_TRANSFORMS_FOO_H`? (In this and other files).